doc/sphinxext/numpydoc/README.rst

```diff
@@ -46,6 +46,8 @@ The following options can be set in conf.py:
   methods and attributes. If a table of contents is made, Sphinx expects
   each entry to have a separate page.
 
-- numpydoc_edit_link: bool  (DEPRECATED -- edit your HTML template instead)
+- numpydoc_edit_link: bool
```

Comment:
please leave this one as is (numpydoc's source is just included in pandas, we should adapt it only minimally)